### DISCUSSION TITLE
How to send an audio file to only one earphone?

### DISCUSSION THREAD

**Discussion by t-09** [jspsych/jsPsych#2345] [Q&A] [answered]
Hi everybody!
I should implement a task in which participants watch stimuli on the screen and simultaneously listen a beep signal either to the right earphone or to the left earphone. Is there a way to modify a plugin to allow an audio file to be presented only in one earphone?
I'm currently using jsPsych v.6.2

Cheers!
Tony

**Comment by jodeleeuw** [comment, marked as answer]
Hi @t-09 ,

There's nothing in the plugins that will let you easily do this. It's possible to configure it using the WebAudio API, but the simpler solution might just be to generate stereo audio files that are only tracked to one ear, and select the appropriate file.

**Reply by t-09** [reply]
Hi Josh! Thanks for the quick response, I'll do as you suggest then!

Cheers
Tony
